ml-proto/README.md

```diff
@@ -152,18 +152,15 @@ The S-expression syntax is defined in `parser.mly`, the opcodes in `lexer.mll`.
 
 ```
 type: i32 | i64 | f32 | f64
-memtype: <type> | i8 | i16
-
+size: 8 | 16 | 32 | 64
 value: <int> | <float>
 var: <int> | $<name>
 
 unop:  neg | abs | not | ...
 binop: add | sub | mul | ...
 relop: eq | neq | lt | ...
-sign: s|u
-align: 1|2|4|8|...
-memop: (<sign>.)?(<align>.)?
-cvtop: trunc_s | trunc_u | extend_s | extend_u | ...
+cvtop: wrap | trunc_<sign> | extend_<sign> | ...
```

Comment:
I guess this was already present, but is there a reason why there are signed and unsigned forms of `trunc` here?